sirepo_bluesky/sirepo_ophyd.py

```diff
@@ -505,4 +524,40 @@ def create_classes(sirepo_data, connection, create_objects=True, extra_model_fie
             if create_objects:
                 objects[object_name] = cls(name=object_name)
 
+            if sim_type == "srw" and model_field == "beamline":
+                prop_params = connection.data["models"]["propagation"][str(el["id"])][0]
+                sirepo_propagation = []
+                object_name += "_propagation"
+                for i in range(9):
+                    sirepo_propagation.append(
+                        SirepoSignal(
+                            name=f"{object_name}_{SimplePropagationConfig._fields[i]}",
+                            value=float(prop_params[i]),
+                            sirepo_dict=prop_params,
+                            sirepo_param=i,
+                        )
+                    )
+                propagation = PropagationConfig(*sirepo_propagation[:])
+                classes[object_name] = propagation
+                if create_objects:
+                    objects[object_name] = propagation
+
+        if sim_type == "srw":
+            post_prop_params = connection.data["models"]["postPropagation"]
+            sirepo_propagation = []
+            object_name = "postPropagation"
```

Comment:
Please see how we did name normalization for the objects ([snake_case](https://github.com/NSLS-II/sirepo-bluesky/pull/130/files/7ad0b07aeb8e8e2abc5d03803d0b8633a394fa9d..6da615ed4ffed38287a48e8b62cdca7206d049e5#diff-8bcde48ffef8d2e24aa6a86e25beac3349a7f5e40c3e3ad41bf3cac096966c8aR453)) vs. classes ([CamelCase](https://github.com/NSLS-II/sirepo-bluesky/pull/130/files/7ad0b07aeb8e8e2abc5d03803d0b8633a394fa9d..6da615ed4ffed38287a48e8b62cdca7206d049e5#diff-8bcde48ffef8d2e24aa6a86e25beac3349a7f5e40c3e3ad41bf3cac096966c8aR448)).